test/integration/render-tests/globe/globe-transition/transition-fill/style.json

```diff
@@ -5,7 +5,7 @@
       "width": 512,
       "height": 512,
       "debug": true,
-      "allowed": 0.004
+      "allowed": 0.03
```

Comment:
Some of the globe render test increases seem a bit high, have you considered moving them to the platform ignores until we have per-platform allowed?